hedera-mirror-rest/monitoring/monitor_apis/account_tests.js

```diff
@@ -265,6 +266,32 @@ const getSingleAccountTokenRelationships = async (server) => {
   };
 };
 
+const getAccountStakingRewards = async (server) => {
+  const stakingRewardsPath = (tokenId) => `${accountsPath}/${stakingRewardAccountId}/rewards`;
```

Comment:
this should be just a string instead of an arrow function
```suggestion
  const stakingRewardsPath = `${accountsPath}/${stakingRewardAccountId}/rewards`;
```

Reply:
Done.